_posts/2016-07-08-confident-freedom-in-community-tgif-91.md

```diff
@@ -0,0 +1,74 @@
+---
+
+title: 'CONFIDENT FREEDOM IN COMMUNITY: TGIF (91)'
+author: Cecily
+layout: blog-post-tgif
+comments: true
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it’s time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. This week, they’re curated for you by [Cecily](http://twitter.com/skeskali). Enjoy!
+
+
+## Open Source
+
+> When people are experiencing hardship in their lives, I try to lessen their burden using refurbished Linux desktops and laptops. I say, let them experience Linux as the face of kindness. For some, they can experience this kindness as a simple act of kindness. For others, it may open new doorways of opportunities in their lives. And then they themselves might choose to offer Linux as the face of kindness. <cite>[Phil Shapiro](https://twitter.com/philshapiro) on [When Linux is the Face of Kindness](https://opensource.com/life/16/7/when-linux-face-kindness)</cite>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Read <a href="https://twitter.com/nithyaruff">@nithyaruff</a>&#39;s open source career advice for women: <a href="https://t.co/Qj49siWaXY">https://t.co/Qj49siWaXY</a> via <a href="https://twitter.com/HuffingtonPost">@HuffingtonPost</a> <a href="https://t.co/AqmGKja7we">pic.twitter.com/AqmGKja7we</a></p>&mdash; The Linux Foundation (@linuxfoundation) <a href="https://twitter.com/linuxfoundation/status/751147313003794432">July 7, 2016</a></blockquote>
+
+[Bulgaria passed a law requiring open source](https://thepolicy.us/bulgaria-got-a-law-requiring-open-source-98bf626cf70a#.bv07px3by)
+
+
+## Design
+
+>Complexity in product design tends to rear its head in two ways 1) the complexity of managing people and opinions. And 2) the complexity of designing the product itself. It’s not always intuitive how to keep your head above water in a sea of features, users and stakeholders. I’ve certainly fallen on my face in the past, so I’d like to share some insights I’ve gleaned about tackling these big design projects. <cite>[Erik Klimczak](https://twitter.com/eklimcz) in [Designing Complex Products](https://blog.truthlabs.com/designing-complex-products-8f9289ab26c9#.ok7ewfgrq)
+
+[Design Better Forms](uxdesign.cc/design-better-forms-96fadca0f49c#.katp81t5k)
+
+[When Everyone Is Doing Design Thinking, Is It Still a Competitive Advantage?](https://hbr.org/2015/08/when-everyone-is-doing-design-thinking-is-it-still-a-competitive-advantage)
+
+
+## The Web and Development
+
+[Inline Validation in Forms - Designing the Experience](https://medium.com/wdstack/inline-validation-in-forms-designing-the-experience-123fb34088ce#.v5uqv15xz)
```

Comment:
When I read it, I felt like it could go in either section, so I put it in Web/Development.